src/dbnode/storage/index_block_test.go

```diff
@@ -847,7 +852,7 @@ func TestLimits(t *testing.T) {
 			if test.requireExhaustive {
 				require.Error(t, err)
 				require.Equal(t, test.expectedErr, err.Error())
-				require.Equal(t, test.expectedInvalidParamsErr, xerrors.IsInvalidParams(err))
+				require.Equal(t, test.expectedQueryLimitExceededError, limits.IsQueryLimitExceededError(err))
```

Comment:
Related to previous comment, it would be best to also keep the check for `InvalidParams` error.